### DISCUSSION TITLE
color_keystrokes: add a space also on the right

### DISCUSSION THREAD

**Discussion by aborruso** [saulpw/visidata#1169] [Ideas]
Hi,
I think it would be better to have a space also on the right of keystrokes, it seems to me more readable.

Thank you

![image](https://user-images.githubusercontent.com/30607/140397099-a0719d38-97ba-48ce-9807-3fab29f253cf.png)

**Comment by anjakefala** [comment]
Moved to issues! https://github.com/saulpw/visidata/issues/1176
